app/views/dashboard/_past_applications.haml

```diff
@@ -5,7 +5,7 @@
       .note_item
         = select_tag :department_ids,
           options_from_collection_for_select(@departments, :id, :name),
-        class: 'dept_select', multiple: true
```

Comment:
I think you should leave this.  I know we don't worry too much about progressive enhancement, but it looks like the plugin will work either way.